observability/pom.xml

```diff
@@ -106,6 +106,10 @@
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-exporter-logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-opentelemetry-exporter-otlp</artifactId>
```

Comment:
I think this one is now included in Quarkus-opentelemetry extension and there's no need to add it anymore to the pom.xml

Reply:
BTW, @jamesnetherton has already removed this jar from pom.xml for 2.14  release

Reply:
Without that artifact, i get `Unrecognized configuration key "quarkus.opentelemetry.tracer.exporter.otlp.endpoint" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo`.

Reply:
Are you sure you're running with 2.14 ?
It seems that this config is still https://quarkus.io/guides/opentelemetry#configuration-reference
otherwise, we need to open an issue for Quarkus

Reply:
Running with CQ `2.13.4.Final`. Can i send this MR to `2.13.x` ?

Reply:
okay that's make sense. Yes go ahead

Reply:
Ok, lets close this one.